src/game/creatures/genCreatures.js

```diff
@@ -231,10 +231,15 @@ const getItems = creatureItemsArray => {
   return items;
 };
 
-const getRandomPos = (chunkSize, left, top) => [
-  Math.floor(Math.random() * chunkSize) + left * chunkSize,
-  Math.floor(Math.random() * chunkSize) + top * chunkSize,
-];
+const getRandomPos = (chunkSize, left, top) => {
+  const x = Math.floor(Math.random() * chunkSize) + left * chunkSize;
+  const y = Math.floor(Math.random() * chunkSize) + top * chunkSize;
+  return {
+    key: () => [x, y].join(),
```

Comment:
11, 4 would give the same key as 1, 14

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#Parameters
> If omitted, the array elements are separated with a comma

Reply:
`11,4 !== 1,14`

Reply:
Oops, my mistake, I thought omitting joined them with no char